packages/eas-build-job/src/__tests__/job.test.ts

```diff
@@ -0,0 +1,42 @@
+import { Workflow } from '../common';
+import { setAndroidBuilderImage, setIosBuilderImageForManagedJob } from '../job';
+
+function testAndroidImage(
+  sdkVersion?: string,
+  reactNativeVersion?: string,
+  expectedImage?: string
+): void {
+  const job: any = { builderEnvironment: {} };
+  setAndroidBuilderImage(job, sdkVersion, reactNativeVersion);
+  expect(job?.builderEnvironment?.image).toBe(expectedImage);
+}
+
+describe(setAndroidBuilderImage, () => {
+  test('selecting image', () => {
+    testAndroidImage('45.0.0', '0.69.0', 'ubuntu-18.04-jdk-11-ndk-r19c');
+    testAndroidImage('45.0.0', '0.67.0', 'ubuntu-18.04-jdk-8-ndk-r19c');
+    testAndroidImage('46.0.0', '0.69.0', 'ubuntu-20.04-jdk-11-ndk-r21e');
+    testAndroidImage(undefined, '0.69.0', undefined);
```

Comment:
this can happen for bare projects that do not use expo, this is technically a change because current implementation would select jdk 11 or 8 based on rn version, but this PR is changing it to always use image tagged with default in turtle code..